### PR TITLE
big update, note branch name

### PR DESCRIPTION
**Pull request by r0bVious**
makes things prettier, debugged the login, improved visual feedback for player gamestate changes, added a loss and win condition

### CODE REVIEW

**Review by j4yxson — APPROVED** [review]
i trust.... if it breaks might be gg

File: map/map.php [hunk 1]
```diff
@@ -287,24 +84,47 @@
...
                 <!-- <img src="money.jpg" /> -->
-                <div class="money-num">80,000</div>
+                <div class="money-num">money change value</div>
             </div>
         </div>
         <h3>click to continue...</h3>
     </div>
 
+    <div id="end-game">
+        <div class="end-game-text">You win. Nice.</div>
```
Comment (CullenMarshall) [inline map/map.php:106]: how can you win?

File: map/map.php [hunk 2]
```diff
@@ -259,19 +56,19 @@
...
             <div class="stat money-container">
                 <label>Money:</label>
-                <span class="player-money-num">100000</span>
+                <span class="player-money-num">100,000</span>
             </div>
         </div>
 
     </div>
 
     <div id="encounter-zone">
```
Comment (CullenMarshall) [inline map/map.php:65]: will battles happen in encounter zone as well?

File: map/map.php [hunk 3]
```diff
@@ -287,24 +84,47 @@
...
+                <!-- <img src="energy.jpg" /> -->
+                <div class="energy-num">end value</div>
+            </div>
+            <div class="attribute drunk">
+                <!-- <img src="drunk.jpg" /> -->
+                <div class="drunk-num">end value</div>
+            </div>
+            <div class="attribute money">
+                <!-- <img src="money.jpg" /> -->
+                <div class="money-num">end value</div>
```
Comment (CullenMarshall) [inline map/map.php:118]: where does this end value come from? 

above this it it 100000, then you changed it to 100,000 

how are you getting this... is it a string or int? maybe I am just dumb and not seeing it

**Review by CullenMarshall — APPROVED** [review]
looks good

**Pull request merged**